composer.json

```diff
@@ -15,6 +15,7 @@
         "phpmyadmin/phpmyadmin": "^4.9"
     },
     "require-dev": {
+        "bennett-treptow/laravel-migration-generator": "^4.3",
```

Comment:
asta nu il pastram. asta era sa il folosesti cat generezi migrari, apoi il scoti afara.. asta ii de executat o singura data, dupa aia ar fi de preferat sa te obisnuiesti tu sa scrii fisiere de migrare in laravel.